README.md

```diff
@@ -64,7 +64,7 @@ The latest version of the AEM CIF Project Archetype, requires the below minimum
 
 | Archetype | AEM 6.4 | AEM 6.5 | Magento                   | CIF Connector | Java |
 | --------- | ------- | ------- | ------------------------- | ------------- | ---- |
-| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 0.10.0        | 1.8  |
+| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 1.0.0       | 1.8  |
```

Comment:
Should be `0.11.0` right? And the old stuff should go into the VERSION.md "archive".

Reply:
I'll update this. I wasn't sure what to write here :)